subgraph/schema.graphql

```diff
@@ -10,8 +10,7 @@ type Auction @entity {
 
```

Comment:
I noticed we dont set the User entity anywhere in the mappings. can we delete this?